src/services/users/index.js

```diff
@@ -152,11 +153,32 @@ const getNewTokens = async (email, currentRefreshToken) => {
 	return { accessToken, refreshToken, userId: user.id }
 }
 
+const getInvitedUserDetail = async (email) => {
+	const result = await  db.query(
+		`SELECT * FROM user_invites
+		WHERE email = $1 
+		`,
+		[email]
+	)
+	const user = result.rows[0]
+	if (!user || user.status === "active") {
```

Comment:
if there's no user, means no such invite exist so maybe a `InvalidInviteError` is more appropriate

Reply:
yes please